### ISSUE TITLE
Align high level simulation study spec with actual scripts available 

### ISSUE THREAD

**Issue by venkatachalapathy** [WheezePuppet/specstar#55]
https://csssaspecsig.slack.com/archives/CGVA2NL8K/p1559930250026200

@will-nehrboss pointed out that the way I framed the simulation study spec needs to be clarified a bit and reframed to respect the way the code works at the moment. 

@WheezePuppet , we can discuss this over at the #analysis channel.

**Comment by divilian** [comment]
Is this issue now obsolete, @venkatachalapathy?

**Comment by venkatachalapathy** [comment]
Yes, I think so. At least it needs to be moved to `phase 2+`  This should be coupled with the experimental design project #30. We should merge the two issues together at some point.
